src/main/java/com/artipie/RepoConfig.java

```diff
@@ -140,14 +149,23 @@ public Vertx vertx() {
      * @return String value.
      */
     private CompletionStage<String> string(final String key) {
-        return this.repo().thenApply(
-            map -> Objects.requireNonNull(
-                map.string(key),
-                String.format("yaml repo.%s is null", key)
+        return this.stringOpt(key).thenApply(
```

Comment:
@olegmoz what does this change have to do with the current task?

Reply:
@carlosmiranda I needed `stringOpt` method, so extracted it from existing `string` method. That's why this change is here

Reply:
@olegmoz all right, thanks for the explanation.